### PR TITLE
exp/services/recoverysigner: remove commands that duplicate tinkey

### PR DESCRIPTION
**Pull request by leighmcculloch**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Remove the `encryption-tink-key` commands that duplicate the same functionality provided by `tinkey`.

### Why

The `tinkey` tool was previously pretty hard to install because we had to download it, install a few dependencies (Java, a specific version of Bazel, a specific version of Python PIP), and then compile it. Both @howardtw and I had issues with this process. @howardtw's computer wouldn't compile it. Mine compiled, but an error with Bazel at the end made it really difficult to find the binary. Overall it was painful to get tinkey. We wrote these commands because we didn't want others to deal with this pain.

However, `tinkey` is getting released binaries and a homebrew bottle as a result of google/tink#375 which will make getting tinkey really easy. The method for installing on Mac and Linux will be as simple as `brew install ...`.

I've tested out the release binary that was shared on that PR, and it indeed is really easy to get now, and really easy to run and use. It does all the things we'd want our commands to do:
- `recoverysigner encryption-tink-keyset create` is replaced with `tinkey create`
- `recoverysigner encryption-tink-keyset encrypt` is replaced with `tinkey convert-key`
- `recoverysigner encryption-tink-keyset decrypt` is replaced with `tinkey convert-key`
- `recoverysigner encryption-tink-keyset rotate` is replaced with `tinkey rotate`

By removing this code we keep our application focused on what its primary goal is. It keeps our code smaller, tests smaller, and we rely on a tool that is developed by some folks who really know what they are doing. We also get the benefits of continued maintenance of tinkey. We can help folks get started by including in our README instructions for how to create an initial tink keyset.

### Known limitations

The tinkey tool still requires Java to be installed, but that's a small thing I think and easy to do, especially one the homebrew bottle is released.

### CODE REVIEW

**Comment by howardtw** [comment]
Have we tested out the output locally to make sure the recovery signer can start with both the encrypted keyset and the cleartext keyset? If we did things differently in our own commands, chances are something can go wrong when we use the output from the tinkey.

**Comment by howardtw** [comment]
I was able to get both the encrypted output and the cleartext output from `tinkey`. However, the formatted as the following:

Encrypted keyset:
```
{
    "keysetInfo": {
        "primaryKeyId": 1742340770,
        "keyInfo": [{
            "typeUrl": "type.googleapis.com/google.crypto.tink.EciesAeadHkdfPrivateKey",
            "outputPrefixType": "TINK",
            "keyId": 1742340770,
            "status": "ENABLED"
        }]
    },
    "encryptedKeyset": "AQICAHgx0+5/aiM0iVW0Pzb7AKDrxOE0BPV8mvmzdY+tAA/nKAH9AfcyUuJMVPSqVJtRp4pmAAABczCCAW8GCSqGSIb3DQEHBqCCAWAwggFcAgEAMIIBVQYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAzZmHOmHn0mRWpSnnYCARCAggEmUGcW9AlE8/UDgUVEbrODWWqn3Vw70LXTAtgjPjTGVXiIFmXWotSHQitaFggXTAZLk7n3L3OPG62jwcSrogqOg88R2Y/sy8ZnezmAC5390NQjU3bAvfurraRLqzGw/ECa6t/UQJvaclRn0j75iiy7p6jQst3ot9Z1ZhMrp+igSx1TxV2ycctNQW6waBThSin44cKSFe0ygdmr7aPxJnxqLc3iNCLDVnF1ze4WuFpXiF+k0IYrLn4pnfym+2YEN1PhVT8XnefDIU1FJRN77UD7LXhqjYHtjPwUI9wUPjOxKrmvHRXVkoO2lBLVVurTl7BHchby6icQeBV2P97oZX1QQ843mOU3ZeTxHMNiiFg8jYnzh1Zrp7LC0F+PsCoHznhyt/VUvg+/"
}
```

Cleartext keyset:
```
{
    "primaryKeyId": 487804444,
    "key": [{
        "keyData": {
            "typeUrl": "type.googleapis.com/google.crypto.tink.EciesAeadHkdfPrivateKey",
            "keyMaterialType": "ASYMMETRIC_PRIVATE",
            "value": "EooBEkQKBAgCEAMSOhI4CjB0eXBlLmdvb2dsZWFwaXMuY29tL2dvb2dsZS5jcnlwdG8udGluay5BZXNHY21LZXkSAhAQGAEYARogX8L4j8Jjhip+zRsAMKlOnCuWbdGZ7lWwbBqTf1B6LD4iIGKRk4VVUhZRisZ/H6A+i3KDAEC0M+1R/75RR00Zym8QGiAaqcm6r9bfgslDZIXUA1Uz4Davln+fKtL2ShQMa7jhhw=="
        },
        "outputPrefixType": "TINK",
        "keyId": 487804444,
        "status": "ENABLED"
    }]
}
```

It won't be easy for users to use these as input to recoverysigner unless we have the recoverysigner take a file as input.

File: exp/services/recoverysigner/README-Unique-Signing-Keys.md [hunk 1]
```diff
@@ -30,20 +30,21 @@ An AWS KMS key can be used to encrypt the Tink keyset.
...
 asymmetric keys.
 
-Create a Tink keyset containing a single key with the create subcommand. The
-private key will be printed to stderr.
+Create a Tink keyset containing a single key with the create subcommand.
+
+Example:
 ```
-$ recoverysigner encryption-tink-keyset create
+$ tinkey create-keyset --out=keyset.json --key-template=ECIES_P256_HKDF_HMAC_SHA256_AES128_GCM
```
Comment (howardtw) [inline exp/services/recoverysigner/README-Unique-Signing-Keys.md:43]: Maybe we also want to mention the [usage](https://github.com/google/tink/blob/master/docs/TINKEY.md#work-with-key-management-system-kms) of  `master-key-uri`.

Comment (leighmcculloch) [reply exp/services/recoverysigner/README-Unique-Signing-Keys.md:43]: Good idea. 76295526e3ade51dd22efd4e489e550ea10eccc2

**Comment by leighmcculloch** [comment]
>It won't be easy for users to use these as input to recoverysigner unless we have the recoverysigner take a file as input.

The JSON output containing new line doesn't prevent the output from being used. I tried it locally and it worked fine both as a command line option and as an environment variable.

It's also not difficult to use `jq` to compact the json if a user really prefers compact JSON. e.g. `cat keyset.json | jq -c`

I had asked about it in google/tink#390, but they made a point about it being easier to read, and after testing it and seeing that it works fine without new lines I don't think it matters now.

**Comment by howardtw** [comment]
> The JSON output containing new line doesn't prevent the output from being used. I tried it locally and it worked fine both as a command line option and as an environment variable.

I'll give it a shot. I'll let you know how it goes.

> It's also not difficult to use jq to compact the json if a user really prefers compact JSON. e.g. cat keyset.json | jq -c

Great tip!

**Comment by howardtw** [comment]
@leighmcculloch I was able to start recoverysigner locally with both an encrypted keyset and a cleartext keyset generated by `tinkey` 🎉

**Pull request merged**